corehq/ex-submodules/couchforms/tests/test_analytics.py

```diff
@@ -165,6 +165,29 @@ def test_get_exports_by_form_es(self):
                     'my://crazy.xmlns/deleted-app']
         }])
 
+        # We still get the form counts when excluding deleted apps
+        self.assertEqual(get_exports_by_form(self.domain, use_es=True, exclude_deleted_apps=True), [{
+            'value': {'xmlns': 'my://crazy.xmlns/', 'submissions': 2},
+            'key': ['exports_forms_analytics_domain', self.app_id_1,
+                    'my://crazy.xmlns/']
+        }, {
+            'value': {
+                'xmlns': 'my://crazy.xmlns/app',
+                'form': {'name': {}, 'id': 0},
+                'app': {'langs': [], 'name': None, 'id': self.app_id_2},
+                'module': {'name': {}, 'id': 0},
+                'app_deleted': False, 'submissions': 1},
+            'key': ['exports_forms_analytics_domain', self.app_id_2,
+                    'my://crazy.xmlns/app']
+        }, {
+            'value': {
+                'xmlns': 'my://crazy.xmlns/deleted-app',
+                'submissions': 1,
+            },
+            'key': ['exports_forms_analytics_domain', self.app_id_3,
+                    'my://crazy.xmlns/deleted-app']
```

Comment:
This is confusing in the context of `exclude_deleted_apps=True`. Can you say a bit more about why this makes sense?

Reply:
Yes, sorry about that - there was a lot of experimentation going on here